cfgov/unprocessed/js/organisms/FilterableListControls.js

```diff
@@ -2,7 +2,12 @@
 
 // Required modules.
 var atomicCheckers = require( '../modules/util/atomic-checkers' );
+var FormValidator = require( 'validate' );
```

Comment:
Yup! Just figure if anything is in both npm and bower it makes sense to use the npm version when possible.